presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -342,7 +342,7 @@ protected Type visitCurrentTime(CurrentTime node, StackableAstVisitorContext<Con
             }
 
             Type type;
-            switch (node.getType()) {
+            switch (node.getOperation()) {
```

Comment:
I somehow feel that `Type` here was better here. What is the motivation behind this rename?

Reply:
I’m adding a proper Type to every IR node, so this name conflicts